src/Raven.Server/Web/System/DatabasesHandler.cs

```diff
@@ -248,6 +238,23 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
             }
         }
 
+        private DynamicJsonValue GetNodeJson(string tag, ServerNode.Role role, LicenseLimits license, ClusterTopology clusterTopology, string database)
+        {
+            var json = new DynamicJsonValue
+            {
+                [nameof(ServerNode.Url)] = GetUrl(tag, clusterTopology),
+                [nameof(ServerNode.ClusterTag)] = tag,
+                [nameof(ServerNode.Database)] = database,
+                [nameof(ServerNode.ServerRole)] = role,
+            };
+
+            if (license != null && license.NodeLicenseDetails.TryGetValue(tag, out var nodeDetails))
+            {
```

Comment:
please add an explanation in the PR description to this change

Reply:
Done